scripts/environments/local.env.example

```diff
@@ -37,12 +37,13 @@ export ENABLE_CSV_AGENT=true
 # This feature will allow users to use the SharePoint connector to ingest documents from folders in SharePoint.
 export ENABLE_SHAREPOINT_CONNECTOR=false
 # If the SharePoint connector is enabled, use the following settings for a list of sharepoint sites and folders you want to index. 
-# If SHAREPOINT_SITES is set to "example.sharepoint.com, example2.sharepoint.com" and
-# SHAREPOINT_FOLDERS is set to "folder1, folder2" then the sharepoint connector will index
-# folder1 from example.sharepoint.com and folder2 from example2.sharepoint.com
-# The position of the folder in the list should match the position of the site in the list.
-export SHAREPOINT_SITES="" # This should be a comma separated list of sharepoint site urls 
-export SHAREPOINT_FOLDERS="" # This should be a comma separated list of sharepoint folders
+# This is a SINGLE QUOTED JSON Object Array containing each sharepoint site and the corresponding entry folder
+# The app will crawl down from the folder specified into subfolders and files
+# Specifying the root folder of "/Shared Documents" will crawl all your documents in your sharepoint site
+export SHAREPOINT_TO_SYNC='[
```

Comment:
This example needs to be in comments and not the actual value. We want the default to be an empty string as this configuration if copied over would result in errors in the Logic App on a default deployment.